plugins/src/test/java/tech/pegasys/pantheon/plugins/TestPantheonEventsPlugin.java

```diff
@@ -62,8 +65,7 @@ private void onBlockAnnounce(final String json) {
     final int blockCount = blockCounter.incrementAndGet();
     LOG.info("I got a new block! (I've seen {}) - {}", blockCount, json);
     try {
-      final File callbackFile =
-          new File(System.getProperty("pantheon.plugins.dir", "plugins"), "newBlock." + blockCount);
+      final File callbackFile = new File(callbackDir, "newBlock." + blockCount);
```

Comment:
Should each plugin get its own directory??

Reply:
I'll do that when we add another plugin.